### DISCUSSION TITLE
What else we need to do for creating the Beta?

### DISCUSSION THREAD

**Discussion by baruchiro** [brafdlog/caspion#132] [General]
So we still have the #50 opened and we still working with the `unifyRepos` branch.

I think we almost finishing the merge, and even if we will not be ready for full production, we will be ready to do some "marketing" actions.

The questions are, what else we need to publish a Beta version, and what our roadmap for marketing, community, publication, and so on?

**Comment by brafdlog** [comment]
Mainly #120  and #85.

I am also doing some work on the progress indication. When I finish with that I will move to the remaining sheets work and then the README.

Do you want to take the remaining sheets? Or something else?

**Reply by baruchiro** [reply]
I think it is better for me to take the sheets and you to take the README.
